### DISCUSSION TITLE
Hardware recommendations

### DISCUSSION THREAD

**Discussion by marksie1988** [blakeblackshear/frigate#12748] [General]
Hi all, 

I'm hoping someone can help me out here, I currently run frigate in LXC container with coral attached, all I have is the reolink doorbell WiFi at the moment and the coral inference is 47ms. 

I plan to add 4x more cameras all 8mp and one 16mp camera. I'm worried that it won't handle it very well and I already get ffmpeg failing a lot from the doorbell camera. 

I think moving to a dedicated device would probably be a good idea, but after forking out a fortune on cameras I'd like to keep costs reasonable on the hardware for frigate without causing too much performance issues. 

What hardware should I be looking at e.g. i3, i5, i7 or alternative. Do I need a good GPU? Or will the coral be sufficient?

Ideally looking for something that's pre built and has M2 as later I'll likely go for the coral M2 storage doesn't matter as I have a NAS where recordings are stored, I may move it local in the future but it's not a priority now unless you think it's a requirement for performance. 

Appreciate any help / insights.

**Comment by NickM-27** [comment]
GPU is highly recommended, most devices with an intel iGPU are sufficient. https://docs.frigate.video/frigate/hardware#server
